packages/web-ui/src/ds/molecules/DocumentTextEditor/editor.tsx

```diff
@@ -81,46 +91,82 @@ export function DocumentTextEditor({
 
   const handleValueChange = useCallback(
     (value: string | undefined) => {
+      setEditorLines(value?.split('\n').length ?? 0)
       onChange?.(value ?? '')
     },
     [onChange],
   )
 
   return (
-    <div className='relative h-full rounded-lg border border-border overflow-hidden'>
-      <div className='absolute top-0 left-0 right-0 bottom-0'>
-        <Editor
-          height='100%'
-          width='100%'
-          theme='latitude'
-          language='document'
-          defaultValue={defaultValue}
-          beforeMount={handleEditorWillMount}
-          onMount={handleEditorDidMount}
-          onChange={handleValueChange}
-          options={{
-            fixedOverflowWidgets: true,
-            lineDecorationsWidth: 0,
-            padding: {
-              top: 16,
-              bottom: 16,
-            },
-            lineNumbers: 'off',
-            minimap: {
-              enabled: false,
-            },
-            copyWithSyntaxHighlighting: false,
-            cursorSmoothCaretAnimation: 'on',
-            occurrencesHighlight: 'off',
-            renderLineHighlight: 'none',
-            tabSize: 2,
-            wordWrap: 'on',
-            readOnly: !!readOnlyMessage,
-            readOnlyMessage: readOnlyMessage
-              ? { value: readOnlyMessage, supportHtml: true }
-              : undefined,
-          }}
-        />
+    <div className='relative h-full rounded-lg border border-border overflow-hidden flex flex-col'>
+      <div className='flex flex-grow relative'>
+        <div className='absolute top-0 left-0 right-0 bottom-0'>
+          <Editor
+            height='100%'
+            width='100%'
+            theme='latitude'
+            language='document'
+            defaultValue={defaultValue}
+            beforeMount={handleEditorWillMount}
+            onMount={handleEditorDidMount}
+            onChange={handleValueChange}
+            options={{
+              fixedOverflowWidgets: true,
+              lineDecorationsWidth: 0,
+              padding: {
+                top: 16,
+                bottom: 16,
```

Comment:
nitpick but can it be '1em'?

Reply:
:(
![image](https://github.com/user-attachments/assets/80279155-cd1d-4ea3-a368-becd7c02b977)

Reply:
Lastima